<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>617</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>26</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/20120225161746_n3fr4 (2).jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnPlay">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="vSliderVolume">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>330</y>
      <width>35</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QSlider 
    {  
        background-color: rgba(255,255,255,0.2);
        min-width:5px; 
        max-width:5px;
        border:15px solid rgba(255,255,255,0.2); 
    }
     QSlider::add-page:vertical
     {      
       
	    background-color: rgb(250, 250, 0);
        width:4px;
     }
     QSlider::sub-page:vertical  
    {
        background-color: rgba(255,255,255,0.5); 
        width:4px;
     }
    QSlider::groove:vertical 
    {
        background:transparent;
        width:6px;
    }
    QSlider::handle:vertical 
    {
         height: 13px;
        width:13px;
 
 margin: -0 -4px; 
	border-image: url(:/image/image/圆形 (10).png);
     }</string>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="hSliderPlayProgress">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>510</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    QSlider::add-page:Horizontal
     {     
background-color: rgba(255, 255, 255,100);
        height:4px;  
     } 
     QSlider::sub-page:Horizontal   
    {  
background-color: rgb(255, 255, 255);
        height:4px;
     }
    QSlider::groove:Horizontal   
    { 
        background:transparent; 
        height:4px;
    }  
    QSlider::handle:Horizontal  
    { 
        height:16px; 
        width:16px; 
	
	border-image: url(:/image/image/圆形 (9).png);
  margin: -6 0px; 
    }
      </string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="lwMusicList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>331</width>
      <height>331</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPreMusic">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>500</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/下一首(底部).png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNextMusic">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>500</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/下一首(底部) (1).png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVolume">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>500</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/音量.png);
QPushButton
{

background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{


background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{

background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblMusicTime">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>510</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;0:0:0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPlayTime">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>510</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;0:0:0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/music_32px_1125557_easyicon.net.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>20</width>
      <height>431</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;网抑云阴乐&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLyrics1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>541</width>
      <height>81</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;欢迎来到QT播放器&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLyrics2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>541</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Welcome to QT mediaplayer&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>btnPlay</zorder>
   <zorder>vSliderVolume</zorder>
   <zorder>hSliderPlayProgress</zorder>
   <zorder>pushButton</zorder>
   <zorder>lwMusicList</zorder>
   <zorder>btnNextMusic</zorder>
   <zorder>btnVolume</zorder>
   <zorder>lblMusicTime</zorder>
   <zorder>lblPlayTime</zorder>
   <zorder>logo</zorder>
   <zorder>line</zorder>
   <zorder>btnPreMusic</zorder>
   <zorder>label</zorder>
   <zorder>lblLyrics1</zorder>
   <zorder>lblLyrics2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>网抑云阴乐</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
